app/articles/server/methods/admin.js

```diff
@@ -0,0 +1,68 @@
+import { Meteor } from 'meteor/meteor';
+import { HTTP } from 'meteor/http';
+import { Random } from 'meteor/random';
+
+import { API } from '../utils/url';
+import { settings } from '../../../settings';
+
+const api = new API();
+
+function setupGhost(user, token) {
+	const rcUrl = Meteor.absoluteUrl().replace(/\/$/, '');
+	const blogTitle = settings.get('Article_Site_title');
+	const blogToken = Random.id(17);
+	const announceToken = `${ blogToken }/${ Random.id(24) }`;
+	const settingsToken = `${ blogToken }/${ Random.id(24) }`;
+	settings.updateById('Announcement_Token', announceToken);
+	settings.updateById('Settings_Token', settingsToken);
+	const data = {
+		setup: [{
+			rc_url: rcUrl,
+			rc_id: user._id,
+			rc_token: token,
+			announce_token: announceToken,
+			settings_token: settingsToken,
+			blogTitle,
+		}],
+	};
+	return HTTP.call('POST', api.setup(), { data, headers: { 'Content-Type': 'application/json' } });
+}
+
+function redirectGhost() {
```

Comment:
Can we make it a `const`, and move it inside the method, as it is not being modified and used outside it?